Sources/AfluentTesting/WaitUntilScheduled.swift

```diff
@@ -9,8 +9,8 @@ import Afluent
 
 extension Task where Failure == Error {
     /// Spawns a new Task to run some async operation and waits for that task to begin execution before proceeding.
-    static func waitUntilScheduled(
-        operation: sending @escaping @isolated(any) () async throws -> Success
+    public static func waitUntilScheduled(
```

Comment:
Let's consider naming. This doesn't actually wait until a task is scheduled, it waits until it starts executing

Reply:
Sure thing, will address this later today.

Reply:
Renamed to `waitUntilExecutionStarted`, how does that sound?

Reply:
Perfect